### DISCUSSION TITLE
🐛 Relative `rome.lspBin` doesn't work in VSCode

### DISCUSSION THREAD

**Discussion by lgarron** [rome/tools#3218] [Suggestions]
### Environment information

```bash
macOS 12.5.1
VSCode v1.71.0
Rome extension v0.14.1
```

### What happened?

In a repo with `rome@next` installed, try to use:

```json5
// .vscode/settings.js
{
  "node_modules/./bin/rome"
}
```

Then "Reload Window" and make sure a `.ts` file is open.

This results in an error: `Rome client: couldn't create connection to server.`
<img width="717" alt="Screen Shot 2022-09-09 at 20 23 02" src="https://user-images.githubusercontent.com/248078/189466868-80acc8be-c72d-4176-a377-4d4b16475d2d.png">

These also don't work: 

- `"./node_modules/./bin/rome"`
- `"../node_modules/./bin/rome"`
- `"node_modules/rome/bin/rome"`
- `"./node_modules/rome/bin/rome"`
- `"../node_modules/rome/bin/rome"`

### Expected result

I'm hoping to configure a relative path, similar to how TypeScript supports:

```json5
// .vscode/settings.js
{
  "typescript.tsdk": "node_modules/typescript/lib"
}
```

Using an absolute path is not an option, given that the repo-specific config needs to work across different environments and architectures (include GitHub Codespaces)

### Code of Conduct

- [X] I agree to follow Rome's Code of Conduct

**Comment by ematipico** [comment]
Hi @lgarron , thank you for reporting the issue. Could you let us know what you're trying to accomplish by changing this setting? 

We usually use this setting only for local development, so knowing that there's some other use case might help us understand how to document the setting and solve the issue

**Comment by lgarron** [comment]
> Hi @lgarron , thank you for reporting the issue. Could you let us know what you're trying to accomplish by changing this setting?
> 
> We usually use this setting only for local development, so knowing that there's some other use case might help us understand how to document the setting and solve the issue

Same as for TypeScript: ensuring that the extension has the same behaviour as any CLI invocations by the project.
Since behaviour can change non-trivially between versions, it would be really handy to use the CLI behaviour for consistent results while developing.

**Comment by lgarron** [comment]
I've run into issues several times at this point, due to a mismatch between the version of Rome installed in `node_modules` for a project, and the version used by the Rome extension. Being able to use the project version from the extension would be a really nice way to save me frustration and debugging time.

**Reply by ematipico** [reply]
I might know what's the cause, feel free to open an issue. It's a feature that we now provide and if it's broken, we need to triage it and fix it.

**Reply by lgarron** [reply]
In this case, it turns out this is due to (under-documented) backwards-incompatible changes in the accepted `rome.json` format. I've resolved those, leaving only https://github.com/rome/tools/issues/3406 for now.
